packages/world/src/modules/keysintable/KeysInTableHook.sol

```diff
@@ -58,68 +59,68 @@ contract KeysInTableHook is IStoreHook {
 
   function onBeforeDeleteRecord(bytes32 tableId, bytes32[] memory key, Schema) public {
     bytes32 keysHash = keccak256(abi.encode(key));
-    (bool has, uint40 index) = UsedKeysIndex.get(tableId, keysHash);
+    (bool has, uint40 index) = UsedKeysIndex.get(UsedKeysIndexTableId, tableId, keysHash);
```

Comment:
does StoreSwitch not work here anymore?

Reply:
I assume it's cause they got namespaced?

Reply:
yeah exactly, it's not using the root namespace anymore but we don't support separate namespaces in the config yet, so need to do it manually here for now by creating a separate table id

Reply:
i think technically it doesn't have to be namespaced though but could continue using the root namespace since we only support installing it as root module for now anyway

Reply:
Reverted this change. Had originally started to refactor the module to support non-root installations but then realized it would be a bigger change because we need to register a store hook on an arbitrary table.